### ISSUE TITLE
No voice feedback for the status change of the button to copy meeting information

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#865]
## Bug Report

**Problematic behavior**
There is the ability to copy the meeting information and the link.

**Expected behavior/code**
When copying the meeting information and/or the meeting link, when using a screen reader, the button state changes to "copied to clipboard" but this information is not announced by VoiceOver.

**Steps to Reproduce**
1. Create a new meeting
2. Tab three times to reach the relevant buttons
3. Activate VoiceOver and validate the buttons to listen to what is announced.

**Possible Solution**

**Additional context/Screenshots**

<img width="404" height="417" alt="" src="https://github.com/user-attachments/assets/9a67aab9-f5d0-4f6c-b6e7-372b314aba5b" />
<img width="404" height="416" alt="" src="https://github.com/user-attachments/assets/c814dc7b-21a4-42a0-8b0e-28bd9a5c2991" />
